frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -267,11 +277,30 @@ export const PaymentInput = (): JSX.Element => {
   )
 }
 
-export const PaymentDrawer = ({
-  paymentsField,
+// Will be extended for stripe unconnected messages too
+const PaymentDisabledMessage = ({
+  isEncryptMode,
 }: {
-  paymentsField: FormPaymentsField
+  isEncryptMode: boolean
 }): JSX.Element | null => {
+  return !isEncryptMode ? (
+    <Box px="1.5rem" pt="2rem" pb="1.5rem">
+      <InlineMessage variant={'info'}>
+        <Text>Payments are not available in email mode forms.</Text>
+      </InlineMessage>
+    </Box>
+  ) : null
+}
+
+type PaymentDrawerProps = {
+  isEncryptMode: boolean
+  paymentsField?: FormPaymentsField
+}
+
+export const PaymentDrawer = ({
+  isEncryptMode,
+  paymentsField,
+}: PaymentDrawerProps): JSX.Element | null => {
```

Comment:
Curious to understand, why do we always specify the return value for Components?

I assume that variables exported from `.tsx` that are in PascalCase are always `<Components />` or `null`

Reply:
I think being more verbose wouldn't hurt?

But I can see why, cause even if we remove the specification the static analyser can still assume the return types too.

@justynoh?

Reply:
no specific reason, and it's pretty standard throughout the codebase. Personally though I like to explicitly type all my functions :D